discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -52,6 +52,26 @@ def _create_tf_folder(self, name: str, platform: str):
 
         return os.path.join(tf_folder, consts.Platforms.BARE_METAL)
 
+    def _get_disk_encryption_appliance(self):
+        if isinstance(self._entity_config, BaseInfraEnvConfig):
+            logging.debug("Infra-env is not associated with any disk-encryption configuration")
```

Comment:
Use `log.debug` instead

Reply:
why? I see this file imports logging and it is being used by multiple functions.
I also don't see any loggers `log` passed around for me to use here

Reply:
Well.. that's a mistake. We should use `test-infra` formatted log and not directly from logging.
I'll open a task to change it to the correct logger.

Reply:
ack. so I'll leave it as-is since it looks out of scope for this PR. Thanks!